### ISSUE TITLE
Demo of router bug.

### ISSUE THREAD

**Issue by nathanhammond** [emberfest/diagonal-routes#7]
This is just me trolling a bit.

http://alexspeller.com/ember-diagonal/?routesInput=this.route(%27epsilon%27%2C%20%7B%20path%3A%20%27%27%20%7D%2C%20function()%20%7B%0A%20%20this.route(%27troll%27)%3B%0A%7D)%3B%0Athis.route(%27index%27%2C%20%7B%20path%3A%20%27%2F%27%20%7D)%3B

/cc @cibernox

**Comment by cibernox** [comment]
At some point this stopped working. Was it fixed?

**Comment by nathanhammond** [comment]
@cibernox No, this is just using an older version I suspect. There have been no changes that would make this work again.
